scripts/lib/build-pgroonga

```diff
@@ -2,12 +2,12 @@
 set -x
 set -e
 
-PGROONGA_VERSION="2.2.3"
+PGROONGA_VERSION="2.2.8"
```

Comment:
Is this version upgrade important for Fedora support?  If so, you should mention that in the commit message.

Reply:
Not important. This version is used in Ubuntu when installed from apt so I matched it here.